app/src/main/res/values-vi/strings.xml

```diff
@@ -73,7 +73,7 @@
     <string name="off_no_salary">Nghỉ không lương</string>
 
     <string name="off_type">Hình thức nghỉ</string>
-    <string name="annual_leave">Nghỉ phép thường niên (14.5 ngày)</string>
+    <string name="annual_leave">Nghỉ phép thường niên (%s ngày)</string>
```

Comment:
WARNING: [StringFormatCount] Inconsistent number of arguments in formatting string `annual_leave`; found both 0 and 1
 When a formatted string takes arguments, it usually needs to reference the same arguments in all translations (or all arguments if there are no translations.

There are cases where this is not the case, so this issue is a warning rather than an error by default. However, this usually happens when a language is not translated or updated correctly.